### PR TITLE
netlink: report neighbors

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Comment by lebauce** [comment]
run scale-tests

File: topology/probes/netlink.go [hunk 1]
```diff
@@ -307,6 +314,52 @@ func (u *NetNsNetLinkProbe) statsToMap(statistics *netlink.LinkStatistics) map[s
...
+}
+
+func (u *NetNsNetLinkProbe) getNeighbors(index, family int) (neighbors []neighbor) {
+	neighList, err := u.handle.NeighList(index, family)
+	if err == nil && len(neighList) > 0 {
+		for _, neigh := range neighList {
+			neighbors = append(neighbors, neighbor{
+				Flags: getFlagsString(neighFlags, neigh.Flags),
+				MAC:   neigh.HardwareAddr.String(),
+				IP:    neigh.IP.String(),
```
Comment (safchain) [inline topology/probes/netlink.go]: you should check that IP is not nil otherwise String return "<nil>"

**Comment by masco-enovance** [comment]
nit:
s/nelink/netlink in commit msg

**Comment by lebauce** [comment]
@masco-enovance Done

**Pull request merged**